pkg/syncer/admissionhandler/cnscsi_admissionhandler.go

```diff
@@ -0,0 +1,99 @@
+package admissionhandler
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	crConfig "sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service/common"
+	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service/logger"
+)
+
+const (
+	//WebhookTlsMinVersion = "1.2"
+	ValidationWebhookPath            = "/validate"
+	DefaultWebhookPort               = 9883
+	DefaultWebhookMetricsBindAddress = "0"
+)
+
+func GetWebhookPort() int {
+	portStr, ok := os.LookupEnv("CNSCSI_WEBHOOK_SERVICE_CONTAINER_PORT")
+	if !ok {
+		return DefaultWebhookPort
+	}
+
+	result, err := strconv.ParseInt(portStr, 0, 0)
```

Comment:
can't we use `strconv.ParseInt(portStr, 10, 64)` and directly return result.

Reply:
yes, we could. This one would also work.